metricbeat/module/kubernetes/state_container/_meta/data.json

```diff
@@ -0,0 +1,45 @@
+{
+  "@timestamp": "2017-05-04T17:46:11.030Z",
+  "beat": {
+    "hostname": "X1",
+    "name": "X1",
+    "version": "6.0.0-alpha1"
+  },
+  "kubernetes": {
+    "container": {
+      "id": "docker://91fdd43f6b1b4c3dd133cfca53e0b1210bc557c2ae56006026b5ccdb5f52826f",
+      "image": "gcr.io/google-containers/kube-addon-manager:v6.3",
+      "name": "kube-addon-manager",
+      "resource": {
+        "requests": {
+          "cpu": {
+            "nanocores": 5000000
+          },
+          "memory": {
+            "bytes": 52428800
+          }
+        }
+      },
+      "status": {
```

Comment:
If we namespace this under `state` it probably becomes more obvious (see data structure comments).